lib/rawHttpRequest/convertArrayFormatToObject.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (targetArray) {
+	return _.reduce(targetArray, (acc, { key, value }) => {
```

Comment:
In the connectors code base, we're moving away from lodash in scenarios where there is a native JS function. Do we want to do the same here, since it looks as though it'll always be an array?

Reply:
I'm not strict about having to use native when possible in the connector platform projects. Either native or lodash is fine.